assets/queries/openAPI/invalid_oauth2_token_url/query.rego

```diff
@@ -0,0 +1,24 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+
+    object.get(doc, "components", "undefined") != "undefined"
+    object.get(doc.components, "securitySchemes", "undefined") != "undefined"
+    sec_scheme := doc.components.securitySchemes[key]
+    sec_scheme.type == "oauth2"
+    url := sec_scheme.flows.authorizationCode.tokenUrl
```

Comment:
I think this should test `password.tokenUrl` and `clientCredentials.tokenUrl` too, since `tokenUrl` is applicable for them too...